pandas/tests/indexes/test_base.py

```diff
@@ -488,21 +492,22 @@ def test_constructor_cast(self):
             Index(["a", "b", "c"], dtype=float)
 
     def test_view_with_args(self):
```

Comment:
did you mean to just remove this test completely? (and replace with one below)

Reply:
the original test has been split